cfgov/v1/models/sublanding_filterable_page.py

```diff
@@ -0,0 +1,66 @@
+from operator import attrgetter
+
+from django.conf import settings
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.db import models
+from django.db.models import Q
+
+from wagtail.wagtailadmin.edit_handlers import StreamFieldPanel, FieldPanel
+from wagtail.wagtailcore.fields import StreamField
+from wagtail.wagtailadmin.edit_handlers import TabbedInterface, ObjectList
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailimages.blocks import ImageChooserBlock
+
+from . import base, molecules, organisms, ref
+from .learn_page import AbstractFilterPage
+from .. import forms
+from ..util.util import get_secondary_nav_items
+
+from .base import CFGOVPage
+
+
+class SublandingFilterablePage(base.CFGOVPage):
+    header = StreamField([
+        ('hero', molecules.Hero()),
+    ], blank=True)
+    sidebar_breakout = StreamField([
+        ('slug', blocks.CharBlock(icon='title')),
+        ('heading', blocks.CharBlock(icon='title')),
+        ('paragraph', blocks.RichTextBlock(icon='edit')),
+        ('breakout_image', blocks.StructBlock([
+            ('image', ImageChooserBlock()),
+            ('is_round', blocks.BooleanBlock(required=False, default=True,
+                                             label='Round?')),
+            ('icon', blocks.CharBlock(help_text='Enter icon class name.')),
+            ('heading', blocks.CharBlock(required=False, label='Introduction Heading')),
+            ('body', blocks.TextBlock(required=False, label='Introduction Body')),
+        ], heading='Breakout Image', icon='image')),
+        ('related_posts', organisms.RelatedPosts()),
+    ], blank=True)
+    content = StreamField([
+        ('full_width_text', organisms.FullWidthText()),
+        ('filter_controls', organisms.FilterControls()),
+    ])
+    secondary_nav_order = models.IntegerField()
+
+    # General content tab
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        StreamFieldPanel('content'),
+    ]
+
+    sidebar_panels = CFGOVPage.sidefoot_panels + [
+        StreamFieldPanel('sidebar_breakout'),
+        FieldPanel('secondary_nav_order'),
+    ]
+
+    # Tab handler interface
+    edit_handler = TabbedInterface([
+        ObjectList(content_panels, heading='General Content'),
+        ObjectList(sidebar_panels, heading='Sidebar'),
+        ObjectList(CFGOVPage.settings_panels, heading='Configuration'),
+    ])
+
```

Comment:
Same with `get_page_set` and `get_form_specific_filter_data`, although you could just call those methods from the BrowseFilterablePage class instead since they don't actually call the object itself...which also make me think maybe it should be pulled out of the class and into utils or something. It's up to you.